@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAttitudeHoldEnable.sqf

```diff
@@ -23,6 +24,11 @@ if (_heli getVariable "fza_ah64_attHoldActive" == false) then {
     //Attitude hold
     if (_curVel > VEL_HOLD_SPEED_SWITCH_ACCEL) then {
         _heli setVariable ["fza_ah64_attHoldSubMode", "att", true];
+
+        private _curAtt   = _heli call BIS_fnc_getPitchBank;
```

Comment:
Why do get the pitch bank, split it out and then rejoin it rather than just using the function output?